.vscode/settings.json

```diff
@@ -1,4 +1,14 @@
 {
   "typescript.tsdk": "node_modules/typescript/lib",
-  "typescript.enablePromptUseWorkspaceTsdk": true
-}
+  "typescript.enablePromptUseWorkspaceTsdk": true,
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "editor.formatOnSave": true,
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+  "tailwindCSS.experimental.classRegex": [
+    ["cva\\(([^)]*)\\)", "[\"'`]([^\"'`]*).*?[\"'`]"]
```

Comment:
V projektu používáme `clsx`, je `cva` lepší, nebo stojí za to změnit tenhle pattern na `clsx`?

Reply:
urcite, kouknu na to. Tohle je pro to, aby to poznalo Tailwind v promennych a doplnovalo intelisense